src/msg/async/EventKqueue.h

```diff
@@ -25,18 +25,33 @@
 
 class KqueueDriver : public EventDriver {
   int kqfd;
-  struct kevent *events;
+  pthread_t mythread;
+  struct kevent *res_events;
   CephContext *cct;
   int size;
 
+  // Keep what we set on the kqfd
+  struct SaveEvent{
+    int fd;
+    int mask;
+  };
+  struct SaveEvent *sav_events;
+  int sav_index;
```

Comment:
Right the idea was to keep save usage here.
Best to name it sav_max and keep the size of sav_events for resizing.